fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -271,6 +273,9 @@ public Response createOrReplaceObjectRdf(
 
         if (httpConfiguration.putRequiresIfMatch() && StringUtils.isBlank(ifMatch) && !resource.isNew()) {
             throw new ClientErrorException("An If-Match header is required", 428);
+        } else if (resource.hasType(FEDORA_PAIRTREE)) {
+            throw new ForbiddenResourceModificationException("Graphs cannot be created for resources " +
```

Comment:
We should probably have symmetry between this line (277) and the previous ticket:
https://github.com/fcrepo4/fcrepo4/pull/829/files#diff-4a14fb4dcbde6f7d32274c43ca723c16R421

I would suggest changing the `ClientErrorException` from the previous ticket to the new `ForbiddenResourceModificationException`.